### PR TITLE
docs: improve configuration chapter

### PR DESCRIPTION
**Pull request by sennierer**

### CODE REVIEW

File: docs/source/configuration.rst [hunk 1]
```diff
@@ -14,3 +14,134 @@ comes from the ``APIS_BASE_URI`` setting. The second part comes from the reverse
 route of ``GetEntityGenericRoot`` if that exists. If not, it uses the
 the reverse route of ``apis_core:GetEntityGeneric``, which is defined in
 :py:mod:`apis_core.urls` and defaults to ``/entity/{pk}``.
+
+Django Settings
+---------------
+
+This section deals with the internal configuration of the APIS tool. For instructions on how to set it up please refer
```
Comment (b1rger) [inline docs/source/configuration.rst:21]:
```suggestion
This section deals with the internal configuration of APIS. For instructions on how to set it up please refer
```

File: docs/source/configuration.rst [hunk 2]
```diff
@@ -14,3 +14,134 @@ comes from the ``APIS_BASE_URI`` setting. The second part comes from the reverse
...
+to :doc:`installation`.
+All the settings described here are part of the Django settings and can be set in the ``settings.py`` file of your Django project.
+
+
+REST_FRAMEWORK
+^^^^^^^^^^^^^^
+
+APIS uses the `Django Restframework`_ for API provisioning. Restframework specific settings like the default page size can be set here.
```
Comment (b1rger) [inline docs/source/configuration.rst:29]:
```suggestion
APIS uses the `Django Restframework <https://www.django-rest-framework.org/>`_ for API provisioning. Restframework specific settings like the default page size can be set here.
```

File: docs/source/configuration.rst [hunk 3]
```diff
@@ -14,3 +14,134 @@ comes from the ``APIS_BASE_URI`` setting. The second part comes from the reverse
...
+
+.. code-block:: python
+
+    REST_FRAMEWORK["DEFAULT_AUTHENTICATION_CLASSES"] = (
+        "rest_framework.permissions.DjangoObjectPermissions",
+    )
+
+Use the above default to allow setting permissions on object level. Meaning that every user gets his permissions depending on his/her user group and the collections this group has permissions for.
```
Comment (b1rger) [inline docs/source/configuration.rst:37]: we don't use django guardian anymore, so we don't have objectlevel permissions, do we?

File: docs/source/configuration.rst [hunk 4]
```diff
@@ -14,3 +14,134 @@ comes from the ``APIS_BASE_URI`` setting. The second part comes from the reverse
...
+
+Sets the default page size the APIS RestAPI should deliver.
+
+
+SPECTACULAR_SETTINGS
+^^^^^^^^^^^^^^^^^^^^
+
+We provide a custom schema generator for spectacular.
+It is meant as a drop in replacement for the
+"DEFAULT_GENERATOR_CLASS" of drf-spectacular. You can use it like this:
```
Comment (b1rger) [inline docs/source/configuration.rst:53]:
```suggestion
``DEFAULT_GENERATOR_CLASS`` of drf-spectacular. You can use it like this:
```

File: docs/source/configuration.rst [hunk 5]
```diff
@@ -14,3 +14,134 @@ comes from the ``APIS_BASE_URI`` setting. The second part comes from the reverse
...
+
+
+APIS_BASE_URI
+^^^^^^^^^^^^^
+
+.. code-block:: python
+
+    APIS_BASE_URI = "https://your-url-goes-here.com"
+
+Sets the base URI your instance should use. This is important as APIS uses mainly URIs instead of IDs. These URIs are also used for the serialization.
```
Comment (b1rger) [inline docs/source/configuration.rst:88]: > These URIs are also used for the serialization.

what does that mean?

File: docs/source/configuration.rst [hunk 6]
```diff
@@ -14,3 +14,134 @@ comes from the ``APIS_BASE_URI`` setting. The second part comes from the reverse
...
+
+APIS_NEXT_PREV
+^^^^^^^^^^^^^^
+
+.. code-block:: python
+    
+    APIS_NEXT_PREV = True
+
+
+APIS_API_ID_WRITABLE
```
Comment (b1rger) [inline docs/source/configuration.rst:99]: This setting does not exist in the codebase anymore (it was part of the old api views)

**Review by b1rger — CHANGES_REQUESTED** [review]

File: docs/source/configuration.rst (hunk 3 above)
Comment (sennierer) [reply docs/source/configuration.rst:37]: goof point, no we dont

File: docs/source/configuration.rst (hunk 5 above)
Comment (sennierer) [reply docs/source/configuration.rst:88]: means that the this is used to generate the URIs used in eg JSON and/or RDF serializations and should be the URL the final production app is running on....rephrased it.

File: docs/source/configuration.rst (hunk 3 above)
Comment (sennierer) [reply docs/source/configuration.rst:37]: removed that and kept ModelPermission

**Review by b1rger — APPROVED** [review]

**Pull request merged**